wizard/src/utils/detect_place.ts

```diff
@@ -194,11 +194,84 @@ export class PlaceDetector {
     return null;
   }
 
+  /**
+   * Country is detected with high confidence if > 90% of the non-null column
+   * values match one of the country format (property) arrays.
+   * If country is not detected, null is returned.
+   * If country is detected, the TypeProperty is returned.
+   *
+   * @param column: an array of strings representing the column values.
+   *
+   * @return the TypeProperty object or null if nothing can be determined with
+   *  high confidence.
+   */
+  detectCountryHighConf(column: Array<string>): TypeProperty {
+    let numValid = 0;
+    let nameCounter = 0;
+    let isoCounter = 0;
+    let alphaNum3Counter = 0;
+    let numberCounter = 0;
+
+    for (const cVal of column) {
+      if (cVal == null) {
+        continue;
+      }
+      const v = toAlphaNumeric(cVal);
+      numValid++;
+
+      if (this.countryNames.has(v)) {
+        nameCounter++;
+      } else if (this.countryISO.has(v)) {
+        isoCounter++;
+      } else if (this.countryAbbrv3.has(v)) {
+        alphaNum3Counter++;
+      } else if (this.countryNumeric.has(v)) {
+        numberCounter++;
+      }
+    }
+
+    // Determine the detected property.
+    let p: DCProperty;
+    if (nameCounter > numValid * MIN_HIGH_CONF_DETECT) {
+      p = this.placeProperties.get("name");
+    } else if (isoCounter > numValid * MIN_HIGH_CONF_DETECT) {
+      p = this.placeProperties.get("isoCode");
+    } else if (alphaNum3Counter > numValid * MIN_HIGH_CONF_DETECT) {
+      p = this.placeProperties.get("countryAlpha3Code");
+    } else if (numberCounter > numValid * MIN_HIGH_CONF_DETECT) {
+      p = this.placeProperties.get("countryNumericCode");
+    } else {
+      p = null;
+    }
+    if (p != null) {
+      return { dcType: this.placeTypes.get("country"), dcProperty: p };
```

Comment:
Updated (2).

Re (1), we can't just to "Country" because this is looking for a DCType object which has two properties. That's what this.placeTypes.get("Country") is returning.